tests/plugins/easyimage/manual/scrollbarchromebug.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: 4.9.0, bug, 932
+@bender-ui: collapsed
+@bender-ckeditor-plugins: sourcearea, wysiwygarea, toolbar, easyimage, link
+@bender-include: ./_helpers/tools.js
```

Comment:
Not sure why this helpers are needed (if for CS think you don't need CS connection here)?

Reply:
Probably I copied entire description from other test and missed it :(